app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -55,6 +56,7 @@ object IntentValidator {
 
     fun validate(context: Context, intent: SafeIntent): ValidatedIntentData? {
         setExperimentOverrides(intent, context)
+        changeFetchDelaySecondsForQA(intent, context)
```

Comment:
nit: -> setQAFetchDelayOverrides for consistency ?

If you were you, I'd prepending "ExperimentOverrides" with QA too.